src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default function App() {
+  return (
+    <div className="app-container">
```

Comment:
Wrap the Display and ButtonPanel in React.Fragment shorthand.
`<>
   ...
</>
`

Reply:
Wrapped the App and ButtonPanel components in React.Fragment as requested. Display has only one node to return, so I did not use React Fragments for Display component.